proto/one/operations/spreadsheet/enum_historian_aggregate_type.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+enum EnumHistorianAggregateType {
```

Comment:
This enum is going to replace existing enum : https://github.com/AquaticInformatics/ONE.Shared.Helpers/blob/main/src/ONE.Shared.Helpers/Historian/HistorianAggregateType.cs